netbox/info.json

```diff
@@ -0,0 +1,2668 @@
+            "url": "",
+            "display": "",
+            "family": {
+              "value": "",
+              "label": ""
+            },
+            "address": "",
+            "vrf": {
+              "id": "",
+              "url": "",
+              "display": "",
+              "name": "",
+              "rd": ""
+            },
+            "tenant": {
+              "id": "",
+              "url": "",
+              "display": "",
+              "name": "",
+              "slug": ""
+            },
+            "status": {
+              "value": "",
+              "label": ""
+            },
+            "role": {
+              "value": "",
+              "label": ""
+            },
+            "assigned_object_type": "",
+            "assigned_object_id": "",
+            "assigned_object": "",
+            "nat_inside": {
+              "id": "",
+              "url": "",
+              "display": "",
+              "family": "",
+              "address": ""
+            },
+            "nat_outside": [
+              {
+                "id": "",
+                "url": "",
+                "display": "",
+                "family": "",
+                "address": ""
+              }
+            ],
+            "dns_name": "",
+            "description": "",
+            "comments": "",
+            "tags": [
+              {
+                "id": "",
+                "url": "",
+                "display": "",
+                "name": "",
+                "slug": "",
+                "color": ""
+              }
+            ],
+            "custom_fields": {
+              "additionalProp1": "",
+              "additionalProp2": "",
+              "additionalProp3": ""
+            },
+            "created": "",
+            "last_updated": ""
+          }
+        ]
+      },
+      "parameters": [
+        {
+          "title": "Order by Field",
+          "name": "ordering",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "text",
+          "tooltip": "Specify the field to use when ordering the results.",
+          "description": "Specify the field to use when ordering the results."
+        },
+        {
+          "title": "offset",
+          "name": "Offset",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "integer",
+          "tooltip": "Specify the initial index from which to return the results.",
+          "description": "Specify the initial index from which to return the results."
+        },
+        {
+          "title": "Limit",
+          "name": "limit",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "integer",
+          "tooltip": "Specify the Number of results to return per page.",
+          "description": "Specify the Number of results to return per page."
+        },
+        {
+          "title": "Other Fields",
```

Comment:
Please add important parameters like id , status, filter etc. if possible in other actions as well